examples/auth/openapi_swagger.json

```diff
@@ -1,21 +1,41 @@
 {
-  "swagger": "2.0",
-  "info": {
-    "description": "Example of the JWT Authn filter using a simple Bookstore API.",
-    "title": "Example - Auth - Bookstore",
-    "version": "1.0.0"
-  },
-  "host": "auth.endpoints.apiproxy-231719.cloud.goog",
   "basePath": "/",
   "consumes": [
     "application/json"
   ],
-  "produces": [
-    "application/json"
-  ],
-  "schemes": [
-    "https"
-  ],
+  "definitions": {
```

Comment:
FYI this sorts the OpenAPI files as well. It's a little weird to see the definitions come before the paths, but I don't have a better solution. I think it should be fine, otherwise it will be hard to maintain. These are small files anyways, so it should not be confusing.